MetaData/data/Era2017_RR-31Mar2018_v1/datasets_3.json

```diff
@@ -1,8 +1,4 @@
 {
```

Comment:
Changes in this file should not be part of this PR. Changes here can be reverted

Reply:
This is fine, it's already fix in the upstream branch the change will go unnoticed